test/errors.ts

```diff
@@ -21,8 +25,7 @@ describe("Error Tests", () => {
         syntheticEvent["lineno"] = lineno;
         syntheticEvent["colno"] = colno;
         errors.logError(syntheticEvent);
-        triggerSend();
-        let events = getAllSentEvents();
+        let events = stopObserving();
         let errorEvents = events.filter((event) => event.state.type === Instrumentation.JsError);
```

Comment:
>let errorEvents = events.filter((event) => event.state.type === Instrumentation.JsError); [](start = 8, length = 89)

can the overserveEvents call now pass Instrumentation.JsError so we don't have to filter here?

Reply:
Yeah, i'll place the condition

---
In reply to: [136674238](https://github.com/Microsoft/clarity-js/pull/28#discussion_r136674238) [](ancestors = 136674238)

Reply:
Actually, you don't need any condition, since all of the new events should be error events only.

---
In reply to: [136677809](https://github.com/Microsoft/clarity-js/pull/28#discussion_r136677809) [](ancestors = 136677809,136674238)